package.json

```diff
@@ -13,13 +13,21 @@
     "lint": "turbo run lint",
     "prepare": "husky install",
     "release": "turbo run build --filter=@formspree/cra-demo^... && changeset publish",
+    "test": "turbo run test",
     "typecheck": "turbo run typecheck",
     "version-packages": "changeset version"
   },
   "devDependencies": {
+    "@babel/core": "^7.22.5",
+    "@babel/preset-env": "^7.22.5",
+    "@babel/preset-typescript": "^7.22.5",
     "@changesets/cli": "^2.22.0",
+    "@types/jest": "^29.5.2",
+    "babel-jest": "^29.5.0",
     "eslint": "^8.15.0",
     "husky": "^8.0.0",
+    "jest": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
```

Comment:
Required as a separate package since Jest 28
https://jestjs.io/docs/28.x/upgrading-to-jest28#jsdom